packages/web/src/scss/helpers/links/index.html

```diff
@@ -1,59 +1,101 @@
-  </h3>
-  <h3 class="typography-heading-xsmall-text">
-    Heading XSmall Text with <a href="#" class="link-primary">link (primary), obtains visited state</a>
-  </h3>
-  <h3 class="typography-heading-xsmall-text">
-    Heading XSmall Text with <a href="#" class="link-underlined">link with underlined class, obtains visited state, gets underlined only on hover and active</a>
-  </h3>
-  <h3 class="typography-heading-xsmall-text">
-    Heading XSmall Text with <a href="#">link without underlined class, behaves the same as the one before</a>
-  </h3>
+  <div class="docs-Stack docs-Stack--start">
+
+    <p>
+      <a href="#" class="link-primary">Primary Link</a>
+    </p>
+    <p>
+      <a href="#" class="link-secondary">Secondary Link</a>
+    </p>
+    <p class="docs-Box">
+      <a href="#" class="link-inverted">Inverted Link</a>
+    </p>
+  
+  </div>
+
+</section>
+
+<section class="docs-Section">
+  
+  <h2 class="docs-Heading">Disabled</h2>
+
+  <div class="docs-Stack docs-Stack--start">
+    <p>
+      <a href="#" class="link-primary link-disabled">Primary Disabled Link</a>
+    </p>
+    <p>
+      <a href="#" class="link-secondary link-disabled">Secondary Disabled Link</a>
+    </p>
+    <p class="docs-Box">
+      <a href="#" class="link-inverted link-disabled">Inverted Disabled Link</a>
+    </p>
+  </div>
+
+</section>
+
+<section class="docs-Section">
+  
+  <h2 class="docs-Heading">Underlined</h2>
+
+  <div class="docs-Stack docs-Stack--start">
+
+    <p>
+      <a href="#" class="link-primary link-underlined">Primary Underlined Link</a>
+    </p>
+    <p>
+      <a href="#" class="link-secondary link-underlined">Secondary Underlined Link</a>
+    </p>
+    <p class="docs-Box">
+      <a href="#" class="link-inverted link-underlined">Inverted Underlined Link</a>
+    </p>
+
+  </div>
 
 </section>
 
-{{/layout/plain}}
+<section class="docs-Section">
+  
+  <h2 class="docs-Heading">Underlined & Disabled</h2>
+
+  <div class="docs-Stack docs-Stack--start">
+    <p>
+      <a href="#" class="link-primary link-disabled link-underlined">Primary Disabled Underlined Link</a>
+    </p>
+    <p>
+      <a href="#" class="link-secondary link-disabled link-underlined">Secondary Disabled Underlined Link</a>
+    </p>
+    <p class="docs-Box mb-1000">
+      <a href="#" class="link-inverted link-disabled link-underlined">Inverted Disabled Underlined Link</a>
+    </p>
+  </div>
+
+</section>
+
+<section class="docs-Section">
+  
+  <h2 class="docs-Heading">Headings with links</h2>
+
+  <div class="docs-Stack docs-Stack--start">
+
+    <h3 class="typography-heading-xsmall-text">
+      Heading XSmall Text with <a href="#">link (default style), obtains visited state</a>
+    </h3>
+    <h3 class="typography-heading-xsmall-text">
+      Heading XSmall Text with <a href="#" class="link-primary">link (primary), obtains visited state</a>
+    </h3>
+    <h3 class="typography-heading-xsmall-text">
```

Comment:
We apparently don't need this demo anymore. So you can remove it, please.

Reply:
removed